Slingshot.Api/Concrete/BitbucketRepository.cs

```diff
@@ -311,5 +338,28 @@ public class OAuthInfo
             public string token_type { get; set; }
             public DateTime expires_at { get; set; }
         }
+
+        public class BitBucketPullRequestSource
+        {
+            public BitBucketPullRequestSourceBranch branch { get; set; }
+        }
+
+        public class BitBucketPullRequestSourceBranch
+        {
+            public string name { get; set; }
+        }
+
+        /// <summary>
+        /// Partial object, full reponse please reference
+        /// https://confluence.atlassian.com/bitbucket/pullrequests-resource-423626332.html#pullrequestsResource-GETaspecificpullrequest
+        /// </summary>
+        public class BitBucketPullRequest : IPullRequestInfo
```

Comment:
Why define fields that we don't use? It's probably simpler to just rely on dymamic to get the things we care about without trying to declare complex data structures